exonum-java-binding/core/rust/build.rs

```diff
@@ -20,6 +20,6 @@ fn main() {
     ProtobufGenerator::with_mod_name("protobuf_mod.rs")
         .with_input_dir("../src/main/proto")
         .add_path("../src/main/proto")
-        .add_path("../../common/src/main/proto/src")
+        .add_path("../../messages/src/main/proto/src")
         .generate();
```

Comment:
@vitvakatu Do we actually use them:

https://github.com/exonum/exonum-java-binding/blob/2ee756cbd09a70db591d7362fdeeffde6476f29e/exonum-java-binding/core/rust/Cargo.toml#L21-L22

?

Reply:
Indeed https://github.com/exonum/exonum-java-binding/pull/1428